apis/fluent-en_GB/extensions/kotlin_1_3/atrium-api-fluent-en_GB-kotlin_1_3-common/src/main/kotlin/ch/tutteli/atrium/api/fluent/en_GB/kotlin_1_3/resultAssertions.kt

```diff
@@ -1,6 +1,31 @@
 package ch.tutteli.atrium.api.fluent.en_GB.kotlin_1_3
 
 import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.domain.builders.ExpectImpl
+import ch.tutteli.atrium.domain.builders.kotlin_1_3.result
 
-//TODO copy KDoc from another assertion which includes a check, e.g. listAssertions.kt -> get
-fun <E, T : Result<E>> Expect<T>.isSuccess(): Expect<E> = TODO("implement")
+/**
+ *
+ * Expects that the given assertion is a success
+ * returns an [Expect]
+ *
+ * @return The newly created [Expect] if the given assertion is success
+ * @throws AssertionError Might throw an [AssertionError] if the given assertion is not a success.
+ *
+ * @since 0.9.0
+ */
+fun <E, T : Result<E>> Expect<T>.isSuccess(): Expect<E> = ExpectImpl.result.isSuccess(this).getExpectOfFeature()
+
+/**
+ *
+ * Expects that the given assertions are success
```

Comment:
Same here, use the above and adapt to the version involving an assertionCreator  (see listAssertions -> get)